apps/worker/src/app/shared/utils/cronHealth.ts

```diff
@@ -0,0 +1,12 @@
+import { Logger } from '@nestjs/common';
+
+const url = 'https://uptime.betterstack.com/api/v1/heartbeat/';
+const LOG_CONTEXT = 'cronHealth';
+export function checkinForCronJob(cronId?: string) {
+  if (process.env.NOVU_MANAGED_SERVICE && process.env.NODE_ENV === 'production' && cronId) {
+    Logger.verbose(`Calling health endpoint for ${cronId}`);
+    fetch(url + cronId)
+      .then((response) => response.json())
```

Comment:
do we need to do something with the response?
looks like we fetch, and then log 'Metric job Completed' Is it the expected behavior?

Reply:
If we don't need the response, maybe we can remove the `.json()` in order to remove the redundant parsing.